euclidean_gcd/C++/euclidean_gcd.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+using namespace std;
+
+int gcd(int a, int b){
+	if (a == 0)
+		return b;
+	return gcd(b % a, a); // returns gcd(remainder, a) if gcd(a,b) = r then gcd(a,b) = gcd(b,r)
+}
+int main(){
+	int a, b;
+	cout << "Enter a and b: " << endl;
+	cin >> a >> b;
+	cout << gcd(a,b);
+	return 0;
+}
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpmkxzxkf4/euclidean_gcd/C++/euclidean_gcd.cpp
+++ b/tmp/tmpmkxzxkf4/euclidean_gcd/C++/euclidean_gcd.cpp
@@ -12,4 +12,4 @@
 	cin >> a >> b;
 	cout << gcd(a,b);
 	return 0;
-}+}
```